extensions/sql-database-projects/src/dialogs/deployDatabaseQuickpick.ts

```diff
@@ -70,6 +71,53 @@ export async function launchDeployAppIntegrationQuickpick(project: Project): Pro
 	};
 }
 
+async function launchEulaQuickPick(baseImage: string): Promise<boolean> {
+	let eulaAccepted: boolean = false;
+	const baseImages = uiUtils.getDockerBaseImages();
+	const imageInfo = baseImages.find(x => x.name === baseImage);
+	const agreementInfo = imageInfo?.agreementInfo;
+	if (agreementInfo) {
+		const openEulaButton: vscode.QuickInputButton = {
+			iconPath: {
```

Comment:
I'd suggest using the built-in link-external icon instead 

![image](https://user-images.githubusercontent.com/28519865/144127396-d7c8f127-123a-4b53-bca9-2c1f7862839c.png)

https://code.visualstudio.com/api/references/icons-in-labels